Signal/src/ViewControllers/ConversationView/ConversationViewController+GestureRecognizers.swift

```diff
@@ -42,13 +42,6 @@ extension ConversationViewController: UIGestureRecognizerDelegate {
 
         // Allow panning with trackpad
         if #available(iOS 13.4, *) { collectionViewPanGestureRecognizer.allowedScrollTypesMask = .continuous }
-
-        // There are cases where we don't have a navigation controller, such as if we got here through 3d touch.
-        // Make sure we only register the gesture interaction if it actually exists. This helps the swipe back
-        // gesture work reliably without conflict with audio scrubbing or swipe-to-repy.
-        if let interactivePopGestureRecognizer = navigationController?.interactivePopGestureRecognizer {
-            collectionViewPanGestureRecognizer.require(toFail: interactivePopGestureRecognizer)
```

Comment:
This priority is inverted. I'm not sure if this used to work or this is just due to the very confusing UIKit API. The pan gesture recognizer should be prioritized over the interactivePop.

I decided against flipping the receiver/argument since it felt weird tweaking settings on a gesture recognizer we don't own. I accomplished something similar with a UIGestureRecognizerDelegate method below.